drivers/dma/dma_stm32_v2.c

```diff
@@ -21,8 +21,12 @@ u32_t table_ll_stream[] = {
 	LL_DMA_CHANNEL_3,
 	LL_DMA_CHANNEL_4,
 	LL_DMA_CHANNEL_5,
-	LL_DMA_CHANNEL_6,
```

Comment:
I think this line should not have been deleted.

Reply:
#if defined(LL_DMA_CHANNEL_6)
	LL_DMA_CHANNEL_6,
#endif /* LL_DMA_CHANNEL_6 */